projects/saturn/src/main/java/io/fairspace/saturn/services/views/ViewService.java

```diff
@@ -81,55 +83,76 @@ public class ViewService {
                FILTER NOT EXISTS { ?subject fs:dateDeleted ?anyDateDeleted }
             }
             """, FS.NS));
+    public static final String USER_DOES_NOT_HAVE_PERMISSIONS_TO_READ_FACETS = "User does not have permissions to read facets";
```

Comment:
personally for me this adds little value, unless there is a global exception message provider, so through the whole application we use the same messages

Reply:
I use the const in test to check the message of thrown exception